### PR TITLE
Add Browse -> Events page

### PR DESCRIPTION
**Pull request by spadgett**
Show recent events for a project in the web console.

events

### CODE REVIEW

**Comment by spadgett** [comment]
@liggitt please review @derekwaynecarr fyi

File: assets/app/scripts/controllers/events.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * Controller of the openshiftConsole
+ */
+angular.module('openshiftConsole')
+  .controller('EventsController', function ($scope, DataService, Logger) {
+    $scope.events = {};
+    var watches = [];
+
+    watches.push(DataService.watch("events", $scope, function(events) {
+      $scope.events = events.by("metadata.name");
+      console.dir($scope.events);
```
Comment (liggitt) [inline assets/app/scripts/controllers/events.js]: remove debug code

File: assets/app/scripts/controllers/events.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * # EventsController
+ * Controller of the openshiftConsole
+ */
+angular.module('openshiftConsole')
+  .controller('EventsController', function ($scope, DataService, Logger) {
+    $scope.events = {};
+    var watches = [];
+
+    watches.push(DataService.watch("events", $scope, function(events) {
+      $scope.events = events.by("metadata.name");
```
Comment (liggitt) [inline assets/app/scripts/controllers/events.js]: do events have labels that would be useful to allow you to filter on? if so, let's hook that up. If not, make sure the label filter widget doesn't show

File: assets/app/scripts/controllers/events.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ * @ngdoc function
+ * @name openshiftConsole.controller:EventsController
+ * @description
+ * # EventsController
+ * Controller of the openshiftConsole
+ */
+angular.module('openshiftConsole')
+  .controller('EventsController', function ($scope, DataService, Logger) {
+    $scope.events = {};
+    var watches = [];
```
Comment (liggitt) [inline assets/app/scripts/controllers/events.js]: initialize emptyMessage to "Loading..." like the other controllers

File: assets/app/views/events.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        <div>
+          <em>{{emptyMessage}}</em>
+        </div>
+      </div>
+      <div class="table-responsive">
+        <table ng-if="(events | hashSize) !== 0" class="table table-bordered table-condensed">
+          <thead>
+            <tr>
+              <th>Time</th>
+              <th>Object</th>
```
Comment (liggitt) [inline assets/app/views/events.html]: split kind and name into their own columns

File: assets/app/views/events.html [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+            </tr>
+          </thead>
+          <tbody>
+            <tr ng-repeat="event in events | toArray | orderBy:'-lastTimestamp'">
+              <td class="nowrap">{{event.lastTimestamp | date:'mediumTime'}}</td>
+              <td>{{event.involvedObject.kind}}/{{event.involvedObject.name}}</td>
+              <td class="nowrap">{{event.reason}}</td>
+              <td>{{event.message}}
+                <span class="text-muted small" ng-if="event.count > 1">
+                  (seen {{event.count}} times in the last
```
Comment (liggitt) [inline assets/app/views/events.html]: maybe drop `seen `?

File: assets/app/scripts/controllers/events.js (hunk 2 above)
Comment (spadgett) [inline assets/app/scripts/controllers/events.js]: Doesn't look like it. I'll remove the filter widget.

**Comment by liggitt** [comment]
Update screenshot, LGTM [merge]

**Comment by spadgett** [comment]
Updated screenshot:

events

**Pull request merged**